parliament/package.json

```diff
@@ -6,47 +6,22 @@
     "type": "git",
     "url": "https://github.com/aol/moloch.git"
   },
-  "scripts": {
```

Comment:
Part of me thinks it's nice to have this for developing. If working from root dir, the git diff's will be several levels deep. But then we are supporting 2 scripts... We could also have the root README scripts just point to the child README scripts.

Reply:
I'm not sure what the best way to do this is. I definitely don't want to support 2 scripts for everything, so I put everything at the top level. @awick what do you think?

Reply:
I say we try it for a bit and if we can't stand it we can add some back that point to the top level like the lint one did.